Userland/Libraries/LibWeb/HTML/Worker.h

```diff
@@ -26,18 +26,18 @@
 namespace Web::HTML {
 
 struct WorkerOptions {
-    DeprecatedString type { "classic" };
-    DeprecatedString credentials { "same-origin" };
-    DeprecatedString name { "" };
+    String type { String::from_utf8("classic"sv).release_value_but_fixme_should_propagate_errors() };
+    String credentials { String::from_utf8("same-origin"sv).release_value_but_fixme_should_propagate_errors() };
+    String name { String::from_utf8_short_string(""sv) };
```

Comment:
Any reason we can't just
```suggestion
    String name {};
```
? I think it's equivalent.

Reply:
None whatsoever, fixed:)